var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -51,7 +51,7 @@ class Extrae(AutotoolsPackage):
     depends_on("elf", type="link")
     depends_on("libxml2")
     depends_on("numactl")
-    depends_on("binutils+libiberty")
+    depends_on("binutils+libiberty@:2.33")
```

Comment:
Would it make sense to condition this on the `extrae` version like:

```
depends_on("binutils+libiberty")
depends_on("binutils+libiberty@:2.33", when="@:3.7.1")
```

? That way, newer `extrae` versions would use a newer binutils

Reply:
Actually this may confuse the concretizer for the time being. SoI think this is good as is.